Assets/Scripts/DataTypes/Car.cs

```diff
@@ -1,33 +1,58 @@
 using UnityEngine;
 
 namespace DataTypes
-{ 
-    public class Car
+{
+    public class Car : GameObjectData<Car, CarBehaviour>
     {
         public Edge road { get; private set; }
         public float positionOnRoad { get; private set; }
         public float lane { get; private set; }
         public float speed { get; private set; } = Conversion.UnitsPerTimeStepFromKPH(50); // Laengeneinheiten pro Zeiteinheit
-        public Transform carTransform;
 
-        public Car(Edge road, float positionOnRoad, float lane)
+        public Car(GameObject prefab, Edge road, float positionOnRoad, float lane) : base(prefab)
         {
             this.road = road;
             this.road.cars.Add(this);
             this.positionOnRoad = positionOnRoad;
             this.lane = lane;
+
+            var position = GetAbsolutePosition();
+            transform.position = new Vector3(position.x, prefab.transform.localScale.y / 2 + prefab.transform.localScale.y / 2, position.y);
```

Comment:
Y-Komponente der Position erzeugt ist falsch, da das Road-Prefab nicht mehr übergeben wird.
Vorschlag: Eine Neue Konstante `ROAD_HEIGHT`, dann lässt sich die y-position wieder berechnen.
```suggestion
            transform.position = new Vector3(position.x, CONSTANTS.ROAD_HEIGHT + prefab.transform.localScale.y / 2, position.y);
```
Dabei ist `CONSTANTS.ROAD_HEIGHT = roadPrefab.localScale.y = 0.05`
Entsprechend muss in `Edge.cs` folgendes gesetzt werden:
```
transform.position.y = CONSTANTS.ROAD_HEIGHT / 2;
transform.localScale.y = CONSTANTS.ROAD_HEIGHT;
```